Documentation/gettingstarted/encryption.rst

```diff
@@ -107,6 +107,11 @@ In order to enable node-to-node encryption, add:
     --set encryption.enabled=true \
     --set encryption.nodeEncryption=true
```

Comment:
Should we add the option here to use direct routing?

I guess (https://docs.cilium.io/en/latest/concepts/networking/routing/#native-routing)  that would be:
```
     --set tunnel=disabled
```

Reply:
The problem sometimes when switching from vxlan to routing its not sufficient to simply disable tunnel, for example in GKE I think you need extra options as well.

But, I guess it makes sense to do this so that it hints to user the requirement.

Also interesting when you tested this what instructions did you follow to bring up a vxlan enabled environment? I thought most of our getting started guides use routing now.

Reply:
> Also interesting when you tested this what instructions did you follow to bring up a vxlan enabled environment? I thought most of our getting started guides use routing now.

I just followed the doc instructions blindly and vxlan was enabled by default.

Reply:
what instructions? The GKE ones should put down a direct routing configuration.

Reply:
These instructions on https://docs.cilium.io/en/v1.9.0-rc2/gettingstarted/encryption/#enable-encryption-in-cilium:

![image](https://user-images.githubusercontent.com/221481/97743533-1ad68a00-1ae6-11eb-95e9-7bb7947b87b5.png)